### ISSUE TITLE
Docker image :stable / :v2.2.7 start failure with docker-compose

### ISSUE THREAD

**Issue by SinErgy84** [paritytech/parity-ethereum#10257]
- **Parity Ethereum version**: v2.2.7 (tag and also stable tag)
- **Operating system**: Linux
- **Installation**: docker
- **Fully synchronized**: yes (with v2.1.11)
- **Network**: ethereum 
- **Restarted**: yes

**actual** 

latest stable tag of the docker image seems to have wrong path configuration

```
Status: Downloaded newer image for parity/parity:v2.2.7
Recreating 25aeef263ffc_parity_stable_ethereum_parity_no_trace_1 ... error

ERROR: for 25aeef263ffc_parity_stable_ethereum_parity_no_trace_1  Cannot start service ethereum_parity_no_trace: b'OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \\"parity\\": executable file not found in $PATH": unknown'
```

**expected behavior**

starting service as expected.

tested downgrade to v2.1.11 stable release - same config - no problem
tested also v2.3.0 beta release - same config - no problem 

**steps to reproduce**

- download docker image `docker pull parity/parity:stable`
- use docker-compose.yml for configuration

```
version: '2'
services:
  ethereum_parity_no_trace:
    image: parity/parity:stable
    volumes:
      - ./volume/:/home/parity/.local/share/io.parity.ethereum/
    entrypoint:
      - parity
      - --no-ipc
      - --jsonrpc-interface
      - "0.0.0.0"
      - --jsonrpc-hosts
      - "all"
      - --jsonrpc-apis
      - "web3,eth,net,personal,rpc,parity_accounts"
    ports:
      - 8547:8545
```

**Comment by joshua-mir** [comment]
https://github.com/paritytech/parity-ethereum/issues/10190#issuecomment-454575182 afaik we don't use the docker files /docker-compose file from the branches to build the actual images... does this also apply to the prebuilt images themselves?

**Comment by SinErgy84** [comment]
@joshua-mir: yes, the described problems occurs with the prebuilt images.

Not tested so far by me: using docker build for own images

**Comment by 5chdn** [comment]
Should be fixed with 2.2.10 coming out later today.
